src/c-writer.cc

```diff
@@ -673,50 +653,45 @@ void CWriter::Write(const LocalName& name) {
   Write(local_sym_map_[name.name]);
 }
 
-std::string CWriter::GetGlobalName(const std::string& name) const {
-  assert(global_sym_map_.count(name) == 1);
-  auto iter = global_sym_map_.find(name);
-  assert(iter != global_sym_map_.end());
-  return iter->second;
-}
-
 void CWriter::Write(const GlobalName& name) {
-  Write(GetGlobalName(name.name));
+  // TODO(keithw) deal with duplicates across module fields
+  assert(global_sym_map_.count(name.name) == 1);
+  auto iter = global_sym_map_.find(name.name);
+  assert(iter != global_sym_map_.end());
+  Write(iter->second);
```

Comment:
Would it be enough to just do `Write(global_sym_map_.at(name.name))` should be enough here.. no need for `count` or `find` maybe?

I guess this is existing code though, so might not make sense as part of this change.

Reply:
Sure, that works fine for now. We can worry about how much defensive programming is worthwhile in the next one.